opm/parser/eclipse/EclipseState/Schedule/Action/ActionContext.hpp

```diff
@@ -50,9 +51,11 @@ class Context {
     void   add(const std::string& func, double value);
 
     std::vector<std::string> wells(const std::string& func) const;
+    const WListManager& wlm() const;
```

Comment:
Could we have a function name that's a little more descriptive than `wlm` please?

Reply:
Ok